### DISCUSSION TITLE
怎么测试cloudflare worker专用ip的速度呢？

### DISCUSSION THREAD

**Discussion by httpsColon** [XIU2/CloudflareSpeedTest#413] [提问]
ip.txt里的地址是哪里来的啊？命令行用什么呢？
CloudflareST.exe -url和CloudflareST.exe -tp用哪个啊

**Comment by XIU2** [comment]
`-f 1.txt` 是指定 IP 数据文件
`-ip 1.1.1.1,2.2.2.2/24` 是直接指定IP地址
